packages/vim.rb

```diff
@@ -1,28 +1,31 @@
 require 'buildsystems/autotools'
+Package.load_package("#{__dir__}/vim_runtime.rb")
 
 class Vim < Autotools
   description 'Vim is a highly configurable text editor built to make creating and changing any kind of text very efficient.'
   homepage 'https://www.vim.org/'
-  version '9.1.0969'
+  version Vim_runtime.version
```

Comment:
Should we put a comment to remind us this package needs rebuilt whenever vim_runtime is updated?

Reply:
We need to modify the updater logic to handle when versions have changed for stuff like this.

`crew update` already handles that... so we need to expand that to our updater.

Leaving that as work to be done.